kw6/stream.py

```diff
@@ -2,7 +2,7 @@
 from xml.dom import minidom
 from pathlib import Path
 from pydantic import BaseModel
-from typing import Optional, List, Dict
+from typing import Dict
 
 from kw6 import Position, PositionHeader, types, settings
 
```

Comment:
Does it make sense to rename the class now since from the outside it doesn't look like a stream anymore?

Maybe?
```
Kw6Reader
```

Reply:
Good point. Does it make more sense to just call it `Reader`? `kw6.Kw6Reader` is somewhat redundant name scoping.

Reply:
It's the name of the class though. If you print the name of the class it's kind of weird if it's just called "Reader" or?

Maybe?
```
from kw6 import Kw6Reader
```

Reply:
On the other hand, doesn't other libraries do it like you said? PIL.Image and np.array?

Reply:
> On the other hand, don't other libraries do it like you said? PIL.Image and np.array?

Yes, and it works fine :slightly_smiling_face: 
```
In [1]: from PIL import Image

In [2]: type(Image.new('RGB', (32, 32)))
Out[2]: PIL.Image.Image
```